### PR TITLE
feature/mf-multigraph-param-geometrymergetype

### PR DESCRIPTION
**Pull request by marionmaiden**
Adding new attribute for multigraph requests that allows custom tiled layer aggregation methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ To perform a release simply do: `mvn clean deploy -DperformRelease=true`
 
 ## Release Notes
 
+### 0.1.9
+
+- Added the multigraph attribute multigraphLayerCustomGeometryMergeAggregation which allows using custom layer aggregation types for tiled serializations. The possible values are defined on MultiGraphGeometryMergeAggregationType.
+
```
Comment (DavidRedlich) [inline README.md]: that's all still part of release 0.1.8

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Get your API key [here](http://targomo.com/developers/pricing/).
      <dependency>
          <groupId>com.targomo</groupId>
          <artifactId>java-client</artifactId>
-         <version>0.1.8</version>
+         <version>0.1.9</version>
```
Comment (DavidRedlich) [inline README.md]: leave 0.1.8

File: pom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
     <groupId>com.targomo</groupId>
     <artifactId>java-client</artifactId>
-    <version>0.1.8-SNAPSHOT</version>
+    <version>0.1.9-SNAPSHOT</version>
```
Comment (DavidRedlich) [inline pom.xml]: same here

File: src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java [hunk 4]
```diff
@@ -199,7 +199,8 @@ private static void addMultiGraphLayer(TravelOptions travelOptions, JSONObject m
         if( Stream.of(travelOptions.getMultiGraphLayerType(), travelOptions.getMultiGraphLayerEdgeAggregationType(),
                 travelOptions.getMultiGraphLayerGeometryDetailPerTile(), travelOptions.getMultiGraphLayerMinGeometryDetailLevel(),
                 travelOptions.getMultiGraphLayerMaxGeometryDetailLevel(),
-                travelOptions.getMultiGraphLayerGeometryDetailLevel()).anyMatch(Objects::nonNull) ) {
+                travelOptions.getMultiGraphLayerGeometryDetailLevel(),
+                travelOptions.getMultigraphLayerCustomGeometryMergeAggregation()).anyMatch(Objects::nonNull) ) {
```
Comment (DavidRedlich) [inline src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java]: you also need to add that in the if statement on line 68

Comment (marionmaiden) [reply src/main/java/com/targomo/client/api/request/config/RequestConfigurator.java]: I have added locally, but as I saw that the other layer attributes weren't there, I removed before committing. Going to add again.

**Pull request merged**